cli/cmd/compliance_gcp.go

```diff
@@ -54,6 +54,10 @@ var (
 				rows = append(rows, []string{gcp.OrganizationID, gcp.ProjectID})
 			}
 
+			if cli.JSONOutput() {
+				return cli.OutputJSON(rows)
```

Comment:
will this be a valid JSON? I thought we needed to fix `[]` this output... 🤔

Reply:
We need to discuss the ux on this. [] is valid json and so is wrapping it in an object {[]}. So we need to find out what the better output is

Reply:
I could not agree more with your, @dmurray-lacework !!! Lets talk about this.

Reply:
@dmurray-lacework could you please update this PR so that the JSON
output matches this structure:
```json
[
  {
    "organization_id": "",
    "project_id": ""
  },
  {
    "organization_id": "",
    "project_id": ""
  }
]
```